src/js/stores/schemes/groupScheme.js

```diff
@@ -0,0 +1,23 @@
+import Util from "../../helpers/Util";
+
+import HealthStatus from "../../constants/HealthStatus";
+
+const groupScheme = {
+  totalCpus: 0,
+  totalMem: 0,
+  instances: 0,
+  health: [
+    {quantity: 0, state: HealthStatus.HEALTHY},
+    {quantity: 0, state: HealthStatus.UNHEALTHY},
+    {quantity: 0, state: HealthStatus.UNKNOWN},
+    {quantity: 0, state: HealthStatus.STAGED},
+    {quantity: 0, state: HealthStatus.OVERCAPACITY},
+    {quantity: 0, state: HealthStatus.UNSCHEDULED}
+  ],
+  tasksStaged: 0,
+  tasksRunning: 0,
+  tasksHealthy: 0,
+  tasksUnhealthy: 0
+};
+
+export default Util.deepFreeze(groupScheme);
```

Comment:
Actually yeah we want this. We are betting on that the health array has the same health status on the same array position so as soon as an application with status Staged is added I do have to add the first four statuses anyway. So I figured why not add them directly.